test/integration/operator_olm_test.go

```diff
@@ -247,3 +248,99 @@ func OperatorPackageManifestBasic(t *testing.T) {
 	// "Remove operator after removal"
 	assert.NoError(t, opcmd.Cleanup())
 }
+
+func OperatorPackageManifestsMultiplePackages(t *testing.T) {
```

Comment:
```suggestion
func TestOperatorPackageManifestsMultiplePackages(t *testing.T) {
```

Reply:
This is a sub-test of `TestOLMIntegration` and should never be run by itself, so I omitted the `Test` prefix. Ditto below.